R/TealAppDriver.R

```diff
@@ -37,16 +37,36 @@ TealAppDriver <- R6::R6Class( # nolint: object_name.
         header = header,
         footer = footer
       )
+
+      dots <- rlang::list2(...)
+      # Default timeout is hardcoded to 4s in shinytest2:::resolve_timeout
+      # It must be set as parameter to the AppDriver
+      if (is.null(dots$timeout)) dots$timeout <- 20 * 1000
+      if (is.null(dots$load_timeout)) dots$load_timeout <- 100 * 1000
```

Comment:
Nice implementation to overwrite the timeouts. I think it's worth mentioning the timeouts in the function doc.

Reply:
![image](https://github.com/insightsengineering/teal/assets/211358/5b678d70-1476-458f-a4a8-dc8e2a14399b)

Does this look good?

Reply:
Thanks for the doc changes. LGTM now :)